azure-pipelines.yml

```diff
@@ -104,23 +104,15 @@ stages:
             inputs:
               mavenPomFile: 'pom.xml'
               goals: 'test'
-              options: $(MVN_OPTS_TEST) -Punit-tests -pl $(JOB1_MODULES),hudi-client/hudi-spark-client
-              publishJUnitResults: false
-              jdkVersionOption: '1.8'
-              mavenOptions: '-Xmx4g'
-          - task: Maven@3.205.1
-            displayName: FT common flink
-            inputs:
-              mavenPomFile: 'pom.xml'
-              goals: 'test'
-              options: $(MVN_OPTS_TEST) -Pfunctional-tests -pl $(JOB1_MODULES)
+              options: $(MVN_OPTS_TEST) -Punit-tests -pl hudi-client/hudi-spark-client
```

Comment:
+1 on saving the CI resources for testing the fix.  Don't forget to revert the changes in yml before merging.